fe/fe-core/src/main/java/org/apache/doris/nereids/glue/LogicalPlanAdapter.java

```diff
@@ -62,21 +65,25 @@ public OutFileClause getOutFileClause() {
         return null;
     }
 
+    public void setResultExprs(List<Expr> resultExprs) {
+        this.resultExprs = resultExprs;
+    }
+
     @Override
     public List<Expr> getResultExprs() {
         return resultExprs;
     }
 
-    public ArrayList<String> getColLabels() {
-        return colLabels;
+    public void setColLabels(ArrayList<String> colLabels) {
+        this.colLabels = colLabels;
     }
 
-    public void setResultExprs(List<Expr> resultExprs) {
-        this.resultExprs = resultExprs;
+    public ArrayList<String> getColLabels() {
```

Comment:
it would be better to keep the origin order of getColLabels and setResultExprs in order to reduce effort of merge code.

Reply:
ok~